lib/main_common.pm

```diff
@@ -1645,6 +1645,7 @@ sub load_extra_tests_console {
     loadtest 'console/aaa_base' unless is_jeos;
     loadtest 'console/libgpiod' if (is_leap('15.1+') || is_tumbleweed) && !(is_jeos && is_x86_64);
     loadtest 'console/osinfo_db' if (is_sle('12-SP3+') && !is_jeos);
+    loadtest 'console/libgcrypt' if (is_sle(">=12-SP3") || is_opensuse);
```

Comment:
For SLE, you can use `if check_var_array('ADDONS', 'sdk') || check_var_array('SCC_ADDONS', 'sdk')`

Reply:
nice, thanks :+1: